server/index.js

```diff
@@ -3,16 +3,10 @@ var express = require('express');
 //middleware
 var morgan = require('morgan');
```

Comment:
I recommend also declaring Morgan and Parser with "const" as well, since you declared cors with "const". Keeps it consistent. Also const is better than var for static variables (or variables that will not change in definition). Utilize "let" for variables with values that will change.

Neat little article stating why var is kind of deprecated: https://medium.com/@pandeysoni/when-should-use-const-and-let-instead-of-var-in-javascript-ec2c3d7e5ca6

Reply:
Nice catch!  Thanks for the review.  Changes have been committed fix this.